servers/cromwell/tox.ini

```diff
@@ -2,6 +2,7 @@
 envlist = py310
 
 [testenv]
+setenv=PIP_CONSTRAINT={toxinidir}/constraints.txt
```

Comment:
I believe it's good practice to include the `constraints.txt` inside a particular `requirements.txt`. I think it could make it a little easier down the road to figure out what is being constrained and why. [This article](https://luminousmen.com/post/pip-constraints-files) seems helpful. Not a huge deal if there's some reason we can't do that.

Reply:
I went with the above per `tox` documentation (here)[https://tox.wiki/en/3.26.0/example/basic.html], which outlines setting constraints as seen above.

Didn't know you could do the above (most Github posts have people adding the `-c <file name>` flag to `pip install`), but seeing how it conflicts with the documented usage in `tox` I've opted for this instead.

That said I do need to update any `pip install` references to include the constraints flag. I'll add a new commit that does it.

Reply:
I'm in favor of doing what `tox` documentation does. 

Good catch on the pip/docker stuff!